community-content/pytorch_pre_built_images_deployment/pytorch-text-classification-vertex-ai-deploy.ipynb

```diff
@@ -0,0 +1,1658 @@
+{
```

Comment:
<p>model.deploy(. -&gt; endpoint = model.deploy</p><p> </p><p>Then you don't need the next cells to get the endpoint</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='90'/>